Create the web3 package structure for 4.x (#4782)
ChainSafe/web3.js: packages/web3/README.md

<p align="center">
  <img src="assets/logo/web3js.jpg" width="500" alt="web3.js" />
</p>

# web3.js

![ES Version](https://img.shields.io/badge/ES-2020-yellow)
![Node Version](https://img.shields.io/badge/node-14.x-green)
[![NPM Package][npm-image]][npm-url]
[![Dependency Status][deps-image]][deps-url]
[![Dev Dependency Status][deps-dev-image]][deps-dev-url]

> This is a sub-package of [web3.js][repo].

Reviewer: ```suggestion
This is the main package of [web3.js](https://github.com/ChainSafe/web3.js).
```
